polyfills/Array/from/polyfill.js

```diff
@@ -1,48 +1,33 @@
 
 // Wrapped in IIFE to prevent leaking to global scope.
 (function () {
-	function parseIterable (arraylike) {
-		var done = false;
-		var iterableResponse;
-		var tempArray = [];
-
-		// if the iterable doesn't have next;
-		// it is an iterable if 'next' is a function but it has not been defined on
-		// the object itself.
-		if (typeof arraylike.next === 'function') {
-			while (!done) {
-				iterableResponse = arraylike.next();
-				if (
-					iterableResponse.hasOwnProperty('value') &&
-					iterableResponse.hasOwnProperty('done')
-				) {
-					if (iterableResponse.done === true) {
-						done = true;
-						break;
-
-					// handle the case where the done value is not Boolean
-					} else if (iterableResponse.done !== false) {
-						break;
-					}
-
-					tempArray.push(iterableResponse.value);
-				} else {
-
-					// it does not conform to the iterable pattern
-					break;
-				}
-			}
+	var mozIterator = '@@iterator';
+	var hasSymbol = typeof Symbol === 'function';
+	var hasMozIterator = typeof [][mozIterator] === 'function';
```

Comment:
This makes it work in Firefox but it will still fail in Safari versions which only support iteration via `for...of` - https://github.com/mathiasbynens/Array.from/blob/master/implementation.js#L168